.travis.yml

```diff
@@ -222,5 +222,12 @@ matrix:
         - tree ./staging
       after_failure:
         - cat "$TRAVIS_BUILD_DIR"/builddir/meson-logs/testlog.txt
+
+    # arm 64
+    - name: arm64
+      os: linux
+      arch: arm64
+      script:
+        - make check
   allow_failures:
```

Comment:
I believe `allow_failures` is part of the `meson` test.

By introducing the `arm64` test in the middle, it presumably transfers the `allow_failures` property from the `meson` test to the `arm64` one.

Reply:
Ah I see. I'll try moving that outside and see what happens.